server/db/Office.js

```diff
@@ -0,0 +1,22 @@
+const { Model } = require('sequelize');
+const reOffice = /[a-z][0-9]{1,2}$/i;
+
+class Office extends Model {
+  static init(sequelize, DataTypes) {
+    const { STRING } = DataTypes;
+    const fields = {
+      name: {
+        type: STRING,
+        allowNull: false,
+        validate: { is: reOffice }
+      }
+    };
+    return super.init(fields, { sequelize });
+  }
+
+  static associate(models) {
+    Office.hasMany(models.User);
```

Comment:
In particular situation:
```js
Office.hasMany(User);
// reads more natural than
Office.hasMany(models.User);
```
so please destructure it directly in argument :wink: